src/network/include/INetworkSerializer.h

```diff
@@ -0,0 +1,50 @@
+#ifndef HIVE_CONNECT_INETWORKSERIALIZER_H
+#define HIVE_CONNECT_INETWORKSERIALIZER_H
+
+#include <common/IProtobufStream.h>
+
+class INetworkSerializer : public IProtobufStream {
```

Comment:
Would it be possible to inject a serializer and network deserializer in a wrapper so read and write are supported?

Reply:
Yes, but we may want to have more than one serializer/client active, while we only want one instance of the server

Reply:
@cquesnel This is why the receive call of the interface is sort of forbidden in the client, and the send on the server